rest-api/src/main/resources/paths/apps/v1_apps_appId_participants_healthCode_versions_version.yml

```diff
@@ -0,0 +1,20 @@
+get:
+    operationId: getParticipantVersionForHealthCode
+    summary: Get participant version for health code.
+    tags:
+        - _For Workers
+    security:
+        - BridgeSecurity: []
+    parameters:
+        - $ref: ../../index.yml#/parameters/appId
+        - $ref: ../../index.yml#/parameters/healthCode
+        - $ref: ../../index.yml#/parameters/version
+    responses:
+        200:
+            description: OK
+            schema:
+                $ref: ../../definitions/participant_version.yml
+        401:
+            $ref: ../../responses/401.yml
+        403:
+            $ref: ../../responses/403_not_worker.yml
```

Comment:
I don't see this in the server PR...was this added earlier?

Reply:
This is an alias for getting participant versions by healthCode. As you note in another comment, we don't want to follow this pattern anymore, so I am deleting this.